<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Downloads/DALL_E_2023-12-17_14.34.39_-_A_humorous_and_unique_interpretation_of_a_dollar_icon._This_icon_features_an_exaggerated__cartoonish_dollar_sign_with_a_quirky_and_fun_expression._Its-removebg-preview.png</normaloff>../../Downloads/DALL_E_2023-12-17_14.34.39_-_A_humorous_and_unique_interpretation_of_a_dollar_icon._This_icon_features_an_exaggerated__cartoonish_dollar_sign_with_a_quirky_and_fun_expression._Its-removebg-preview.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>281</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
background-color: qlineargradient(spread:pad, x1:0.715909, y1:0.381, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string> CURRENCY CONVERTER</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="basevalue_lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="targetvalue_lineEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="base_comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>EUR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>USD</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>GBP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TRY</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="target_comboBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>150</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>EUR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>USD</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>GBP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TRY</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="currencyrate_lineEdit">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>200</y>
      <width>60</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>124</x>
      <y>150</y>
      <width>60</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>RATE</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>380</y>
      <width>1290</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Downloads/DALL·E 2023-12-17 14.52.09 - A straightforward image of a classic bank door, open and inviting. The door is traditional in style, made of polished wood or metal, commonly found in.png</normaloff>../../Downloads/DALL·E 2023-12-17 14.52.09 - A straightforward image of a classic bank door, open and inviting. The door is traditional in style, made of polished wood or metal, commonly found in.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>1000</width>
      <height>1000</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>-75</x>
      <y>10</y>
      <width>820</width>
      <height>450</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Downloads/background.png</normaloff>../../Downloads/background.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>700</width>
      <height>700</height>
     </size>
    </property>
   </widget>
   <zorder>pushButton</zorder>
   <zorder>label</zorder>
   <zorder>basevalue_lineEdit</zorder>
   <zorder>targetvalue_lineEdit</zorder>
   <zorder>base_comboBox</zorder>
   <zorder>target_comboBox</zorder>
   <zorder>currencyrate_lineEdit</zorder>
   <zorder>label_2</zorder>
   <zorder>pushButton_6</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
